frontend/src/server/api/routers/user.ts

```diff
@@ -143,11 +143,11 @@ export const userRouter = createTRPCRouter({
       );
     }
   }),
-  getUsersByIds: protectedProcedure
+  getUsersByIds: publicProcedure
     .input(z.object({ ids: z.array(z.string()) }))
     .query(async ({ input, ctx }) => {
-      const user = ctx.session.user;
-      const apiKey = user.apikey;
+      const user = ctx.session?.user;
+      const apiKey = user?.apikey ?? env.SYS_ADMIN_API_KEY;
```

Comment:
Shouldn't you send an empty key when the user is not signed in to return the public data from the CKAN?

Reply:
thanks for spotting this, this is no longer needed cause of the backed changes